### ISSUE TITLE
Build script

### ISSUE THREAD

**Issue by martijndierckx** [shelfio/libreoffice-lambda-layer#39]
Did you use a specific build script for Libre Office?
I'm wondering if using a new Libre Office version would for example solve the issue where we need to start soffice.bin twice.

**Comment by vladgolubev** [comment]
Yes, here https://github.com/vladgolubev/serverless-libreoffice/blob/master/Dockerfile
